source/common/modules/markdown-editor/plugins/render-images.ts

```diff
@@ -1,202 +1,227 @@
+
+/**
+ * Defines the CodeMirror command to render all found markdown images.
+ * @param  {CodeMirror.Editor} cm The calling CodeMirror instance
+ * @return {void}    Commands do not return.
+ */
+;(commands as any).markdownRenderImages = function (cm: CodeMirror.Editor): void {
+  // We'll only render the viewport
+  const viewport = cm.getViewport()
+  for (let i = viewport.from; i < viewport.to; i++) {
+    if (cm.getModeAt({ line: i, ch: 0 }).name !== 'markdown-zkn') {
+      continue
+    }
+
+    // Always reset lastIndex property, because test()-ing on regular
+    // expressions advance it.
+    imageRE.lastIndex = 0
+
+    // First get the line and test if the contents contain an image
+    let line = cm.getLine(i)
+    if (!imageRE.test(line)) continue
+
+    imageRE.lastIndex = 0 // Necessary because of global flag in RegExp
+
+    let match
+
+    // Run through all links on this line
+    while ((match = imageRE.exec(line)) != null) {
+      // The image RE will give us the following groups:
+      // p1: The alternative text (in square brackets)
+      // p2: The complete contents of the round braces
+      // p3: If applicable, an image title (within round braces)
+      // p4: Anything in curly brackets (mostly commands for Pandoc)
+      let altText = match[1] ?? '' // Everything inside the square brackets
+      let url = match[2] ?? '' // The URL
+      let title = match[3] ?? altText // An optional title in quotes after the image
+      let p4 = match[4] ?? ''
+
+      let widthAttribute = ''
+      let heightAttribute = ''
+      if (match[4] !== '') {
+        // Use regex to parse pandoc attributes
+        // like width="3.69in" height="1.93884in"
+        let widthRE = /(width=)"([^"]+)"/g
+        let heightRE = /(height=)"([^"]+)"/g

```

Comment:
```suggestion
        const widthRE = /(?:width=)"([^"]+)"/g
        const heightRE = /(?:height=)"([^"]+)"/g
```

This makes the first group non-capturing, i.e. below you can access them with `mat[1]` instead.